lib/modules/errors/error_handler.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "./error_serializer.rb"
+
+module ErrorHandler 
+  def route_not_found
+    detail_message = params[:action].split("_").join(" ")
+    status_code = 400
+    source_info = request.original_fullpath
+    render json: ErrorSerializer.bad_request(detail_message, status_code, source_info), status: :bad_request
```

Comment:
This is well thought out! However, do we want to include a message that may be easier to read for the users when they reach this?

Reply:
This is for the FE devs as they will be our app's clients. Also, the serializer's default title and details will be human readable.

Reply:
Further, note that this error should not occur more than when the api call is being set up (in theory) as, once the FE sets up the call correctly they should never hit it.